TROUBLESHOOTING.md

```diff
@@ -1,7 +1,70 @@
-# Performance Caveats
+# Debugging
+
+Note that the information in this section is subject to be removed in future releases of
+the _PyTorch/XLA_ software, since many of them are peculiar to a given internal implementation
```

Comment:
I'm not sure about our use of italic here and in other places, not sure what criteria we're using to apply it. If we want to use it for proper nouns / project names, we should reserve it for that. Also, given that we reference the name of this project and adjacent ones (PyTorch, XLA) quite often I'm not sure this level of emphasis looks good in the final result. This doesn't need to be addressed right now, but in an ideal world we should be consistent with the style of PyTorch documentation - how does it handle other project names?

Reply:
I switched to quote for these notices which is never used so far ;)
I don't think Pytorch has explicit guideline in writing markdown docs yet... (Or maybe I'm not aware of it since I didn't do it much).

Reply:
I agree for consistency, but if we want to stick to pytorch style (they seem to not use italic for things like Python or other project/tools names) we need to change all our .md files.
I suggest maybe get a tech writer input, and defer this change to a further PR.

Reply:
Sure I'll just leave this sentence plain (since there's italic inside it) and we can come back to it later.